spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubOperations.java

```diff
@@ -75,6 +78,17 @@
 	 */
 	ListenableFuture<String> publish(String topic, ByteString payload, Map<String, String> headers);
 
+	/**
+	 * Send a message to Pub/Sub.
+	 * @param topic the name of an existing topic
+	 * @param payload an object that will be serialized and sent
+	 * @param headers map of String to String headers
+	 * @return the listenable future of the call
+	 * @throws JsonProcessingException when the payload cannot be serialized into JSON
```

Comment:
No this exception. You can't be sure that we always use only JSON encoding/decoding.

Reply:
removed.